src/xml/SVGFile.java

```diff
@@ -0,0 +1,155 @@
+package xml;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import model.Canvas;
+import model.Shape;
+import model.specific_path.*;
+import view.Presentation;
+import view.ShapeState;
+import xml.*;
+
+public class SVGFile implements Presentation{
```

Comment:
Not sure that we should remove that class. I do agree for Path, but not really for this one, as it gives the xml part for the global image. (Which you put in model, or the presentation package xml should be the only place where we should find xml)